sources/packages/api/src-sql/Types/Create-restriction-action-type.sql

```diff
@@ -0,0 +1,7 @@
+-- Create an enum for restriction action type.
+CREATE TYPE sims.restriction_action_type AS ENUM (
```

Comment:
The types that we creates so far have the "s" in the end, as shown below. Can we change it to be aligned with the existing ones?
![image](https://user-images.githubusercontent.com/61259237/168680381-0bd88394-69ce-452f-8a0f-a550bba72e43.png)

Reply:
May be we can add this point to wiki to make have us all in same page.